README.rst

```diff
@@ -34,6 +34,14 @@ In the Kinto project settings
     kinto.attachment.folder = {bucket_id}/{collection_id}
     kinto.attachment.keep_old_files = true
 
+
+If you want uploaded files to get gzipped when stored:
+
+.. code-block:: ini
+
+    kinto.attachment.gzipped = true
```

Comment:
For instance if you want to store fonts and images on the same kinto, you will probably want to gzip the fonts but not the images.

That's a use case we have on kinto-ota for instance.